test/utils/utils.ts

```diff
@@ -303,27 +309,55 @@ export async function buildDepositStruct(
     blockNumber: await getLastBlockNumber(),
   };
 }
+
+// export function buildV3Deposit(
```

Comment:
building a v3 deposit struct will be very useful when integrating into relayer-v2, did you just want to add in a followup?

Reply:
Yeah - we'll need the new ABI for that so it's not included here, but it will follow shortly.

Reply:
PR for adding the new ABI here: https://github.com/across-protocol/sdk-v2/pull/526

But since we also need to deploy and run the contracts in a hardhat environment, we'll also need a mocked SpokePool instance (or a separate copy of the updated contracts-v2 repo).